modules/meeting/app/components/meeting_sections/backlogs/header_component.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module MeetingSections
+  class Backlogs::HeaderComponent < ApplicationComponent
+    include ApplicationHelper
+    include OpTurbo::Streamable
+    include OpPrimer::ComponentHelpers
+
+    def initialize(backlog:, collapsed:, box: nil)
+      super
+
+      @backlog = backlog
+      @meeting = backlog.meeting
+      @box = box
+
+      @collapsed = collapsed.nil? ? default : collapsed
+    end
+
+    private
+
+    def default
+      if @meeting.open?
+        false
+      elsif @meeting.in_progress?
+        true
+      end
+    end
```

Comment:
```suggestion
def default
  @meeting.in_progress?
end
```

If I'm not mistaken, you only want it to be open if in progress? Also, I would rename this method to `default_collapsed_state` to make it clearer.